templates/index.html

```diff
@@ -11,7 +12,7 @@
         </nav>
     </header>
     <div>
-      <wpt-results></wpt-results>
+      <wpt-results test-runs="{{ . }}"></wpt-results>
```

Comment:
The `[]TestRun` JSON from the handler gets inserted directly into this element's property